google/cloud/forseti/services/cli.py

```diff
@@ -87,7 +87,11 @@ def define_inventory_parser(parent):
         help='Purge all inventory data older than the retention days.')
     purge_inventory_parser.add_argument(
         'retention_days',
-        help='Number of days to retain the data.')
+        default=None,
```

Comment:
Is the default value None necessary? I think if you put a default value = None here it might get converted to a string "None" which will cause problem in the if statement since (not "None") is false.

Reply:
This will stays None (i.e. null) even when stuffed into proto string, so this will be handled correctly.

Reply:
It's not really needed... but I like that's explicitly set to None here.